ecaudit/src/test/java/com/ericsson/bss/cassandra/ecaudit/config/TestAuditYamlConfigurationLoader.java

```diff
@@ -76,6 +93,10 @@ public void testMissingWhitelistIsDefault()
 
         assertThat(config.getYamlWhitelist()).isEmpty();
         assertThat(config.isPostLogging()).isFalse();
+        assertThat(config.getBoundValueSuppressor()).isEqualTo(SuppressNothing.class.getName());
+        assertThat(config.getWhitelistCacheValidity()).isEqualTo(DatabaseDescriptor.getRolesValidity());
+        assertThat(config.getWhitelistCacheUpdateInterval()).isEqualTo(DatabaseDescriptor.getRolesUpdateInterval());
+        assertThat(config.getWhitelistCacheMaxEntries()).isEqualTo(DatabaseDescriptor.getRolesCacheMaxEntries());
```

Comment:
Verify these in separate tests? Or at least rename the test case.

By default whitelist_max_entries should be ten times roles_max_entries.

Reply:
renamed to testDefaultConfiguration and testCustomConfiguration